### ISSUE TITLE
OSError: 'seaborn-colorblind' is not a valid package style

### ISSUE THREAD

**Issue by 1kastner** [1kastner/conflowgen#197]
**Describe the bug**
```
OSError: 'seaborn-colorblind' is not a valid package style, path of style file, URL of style file, or library style name (library styles are listed in `style.available`)
```

**To Reproduce**

Invoke `<MatPlotLibPreview/MatPlotLibAnalysis>.show_report_as_graph()`

**Proposed Solution**
Use a different default style.

**Comment by 1kastner** [comment]
A new version of ConFlowGen needs to be published - to be done soon

**Comment by 1kastner** [comment]
solved with ConFlowGen v2.1.1.
